src/AbstractFixer.php

```diff
@@ -97,6 +91,13 @@ public function getDefinition()
         );
     }
 
+    /**
+     * Legacy, will be removed if proper description has been added to all fixers.
+     *
+     * @return string
+     */
+    abstract protected function getDescription();
```

Comment:
Everywhere there is `ShortDefinition` thanks to that `AbstractFixer` implementation.
Don't focus on making full `Definition` for all 150 fixers in this PR, we could do this after 2.0 release